leftwm/src/bin/leftwm-worker.rs

```diff
@@ -14,10 +14,15 @@ fn main() {
         let rt = tokio::runtime::Runtime::new().expect("ERROR: couldn't init Tokio runtime");
         let _rt_guard = rt.enter();
 
+        #[cfg(feature = "lefthk")]
         let mut config = leftwm::load();
         // Clear the keybinds so leftwm is not storing them.
         // TODO: Make this more elegant.
```

Comment:
```suggestion
```
or do we want this even more elegant?

Reply:
even more elegant, preferably when we redo the config setup we will have something like:
```
pub struct Config {
    pub  settings: Settings,
    pub keybinds: Keybinds,
}
```
Then Settings will implemented the leftwm config trait and keybinds will implement lefthk config trait, then settings gets passed to leftwm and the keybinds to lefthk after loading the config